addons/inkgd/editor/ink.gd

```diff
@@ -10,15 +10,44 @@ extends EditorPlugin
 var dock = null
 
 func _enter_tree():
-    dock = preload("res://addons/inkgd/editor/ink_dock.tscn").instance()
-    add_control_to_dock(DOCK_SLOT_RIGHT_UL, dock)
+	dock = preload("res://addons/inkgd/editor/ink_dock.tscn").instance()
+	add_control_to_dock(DOCK_SLOT_RIGHT_UL, dock)
+	add_autoloads()
+	add_templates()
+	get_editor_interface().get_resource_filesystem().scan()
 
 func _exit_tree():
-    # Remove from docks (must be called so layout is updated and saved)
-    remove_control_from_docks(dock)
-    # Remove the node
-    dock.free()
+	# Remove from docks (must be called so layout is updated and saved)
+	remove_control_from_docks(dock)
+	# Remove the node
+	dock.queue_free()
+	remove_autoloads()
+	remove_templates()
+	get_editor_interface().get_resource_filesystem().scan()
 
 func build():
-    dock._build_button_pressed()
-    return true
+	dock._build_button_pressed()
+	return true
+
+func add_autoloads():
+	#Find the Ink runtime code and add it as a singleton.
```

Comment:
Putting a space between the comment delimiter and the content makes them more readable IMO!
(Same below and elsewhere.)